File: StykerDG.FarmForge.DataModel/Models/CropClassification.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace StrykerDG.FarmForge.DataModel.Models
{
    [Table("CropClassification")]
    public class CropClassification : BaseModel
    {
        [Key]
        public int CropClassificationId { get; set; }
        public string Name { get; set; }
        public string Label { get; set; }
        public string Description { get; set; }
    }
}

File: StrykerDG.FarmForge.Client/farmforge_client/lib/models/crops/crop_classification.dart
class CropClassification {
  int cropClassificationId;
  String name;
  String label;
  String description;

  CropClassification({
    this.cropClassificationId,
    this.name,
    this.label,
    this.description
  });

  factory CropClassification.fromMap(Map<String, dynamic> data) {
    return CropClassification(
      cropClassificationId: data['CropClassificationId'],
      name: data['Name'],
      label: data['Label'],
      description: data['Description']
    );
  }
}